content/reference/user/overview/index.md

```diff
@@ -1,28 +1,41 @@
 ---
 title: "Overview"
 description: "Learn how to navigate."
-lead: "FrontLine navigation"
+lead: "Gatling Enterprise Overview"
 date: 2021-03-10T09:29:04-05:00
 lastmod: 2021-03-10T09:29:04-05:00
 weight: 10020
 ---
 
-Once you are logged in, you are now able to navigate using the FrontLine navigation bar on the left side.
+{{< img src="overview.png" alt="Overview" caption="Overview" >}}
 
-{{< img src="navbar.png" alt="Navbar" >}}
+## Navigation Bar
 
-FrontLine is composed of:
+Gatling Enterprise navigation bar on the left side can be deployed using the Burger Navigation Menu button. {{< icon bars >}}
 
-- **Profile**, manage account settings
-- **Users**, manage users information, teams and permissions - *accessible to system admin*
-- **Teams**, manage teams - *accessible to system admin*
-- **API Tokens**, manage token to access the API - *accessible to system admin*
-- **Artifacts**, manage simulations artifacts - *accessible from test admin*
-- **Simulations**, manage simulations, runs and reports - *accessible from test admin*
+Gatling Enterprise navigation bar is composed of:
 
-- **Plugins**, displaying all plugins downloads links
+- [**Users**]({{< ref "../../admin/users" >}}), manage users teams and roles - *accessible to system admin*
+- [**Teams**]({{< ref "../../admin/teams" >}}), manage teams - *accessible to system admin*
+- [**API Tokens**]({{< ref "../../admin/api_tokens" >}}), manage token to access the API - *accessible to system admin*
+- [**Artifacts**]({{< ref "../artifact_conf" >}}), manage simulations artifacts - *accessible from test admin*
+- [**Simulations**]({{< ref "../simulations" >}}), manage simulations, runs and reports - *accessible from viewer*
+[fixme]: dedicated ips ref
+- **Dedicated IP's**, table of pools dedicated IP's- *accessible from test admin* 
```

Comment:
There's an odd mix of "accessible to" and "accessible from". If the intention is to use "from" to mean 'including higher privilege roles", it's not obvious at all. I'd use something more explicit, like for instance:

- "accessible to system admins only"
- "accessible to test admins and system admins"
- "accessible to anyone with at least a viewer role"